frontend/components/modals/channel_form_modal.jsx

```diff
@@ -34,8 +35,13 @@ class ChannelFormModal extends React.Component {
     this.props.formAction(this.state, this.props.selectedUsers);
   }
 
+  disabledStatus(){
+    if (!this.state.name) return true;
+    return this.state.private && Object.keys(this.props.selectedUsers).length == 0;
+  }
```

Comment:
this line is a little long. either destructure stuff before to make it shorter, or make it multi-line